### ISSUE TITLE
[004] - [AdminPanelUI] - мерж с ServerSide

### ISSUE THREAD

**Issue by HollowYourDreams** [grishinrv/SCS#31]
- [ ] Подписаться на события Hub'a
- [ ] Смержиться с master
- [ ] Изменить Модель данных

**Comment by grishinrv** [comment]
Предлагаю протестировать с тестовым локером (консолькой) и рабочей админкой. Потом можно протестить с рабочим локером, если все ок.

**Comment by grishinrv** [comment]
Насколько я вижу, в app.config админки нет appId и apikey. Необходимо добавить и заинсертить в БД (воспользоваться SecreteKeyGenerator , только заменить WorksationName и ClientAppGroupId на '83b1ab62-ad7c-4a2e-8118-b4c1571060f1')

**Comment by grishinrv** [comment]
Так-же в app.config необходимо добавить ключ для обращения к серверу (пример в тестовых консольных прилжухах)
